skiko/src/awtMain/kotlin/org/jetbrains/skiko/SkiaLayer.awt.kt

```diff
@@ -326,6 +324,16 @@ actual open class SkiaLayer internal constructor(
         }
     }
 
+    override fun setBounds(x: Int, y: Int, width: Int, height: Int) {
+        super.setBounds(x, y, width, height)
+
+        // To avoid visual artifacts (especially on Windows/DirectX),
+        // redrawing should be performed immediately, without scheduling to "later".
+        // Subscribing to events instead of overriding this method won't help too.
+        redrawer?.syncSize()
+        redrawer?.redrawImmediately()
```

Comment:
```suggestion
        if (isRendering) {
            redrawer?.needRedraw()
        } else {
            redrawer?.redrawImmediately()
        }
```

in case someone will call `setBounds` in a composition. We had reports in the past, that is why we have this in `paint`.